cypress/integration/sidebar-navigation/index.spec.ts

```diff
@@ -61,48 +59,3 @@ describe('Sidebar opens correct section on first load', () => {
     alertDialogLink.should('have.attr', 'aria-current', 'page');
   });
 });
-
```

Comment:
So this was a way that would, after each item was open, try and keep track if it had an open parent and only close those so how, so that if it was a new top level item it would start closed and ensure the expanded attribute was set correctly. 

I figured it'd just be easier to open them all, then reverse the order and close them all. Proves the same outcome, just way less code.